src/imgui/ImGuiReverseBar.cc

```diff
@@ -190,22 +190,52 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 				ImGui::TextUnformatted("No replays found"sv);
 			} else {
 				ImGui::TextUnformatted("Select replay"sv);
-				im::ListBox("##select-replay", [&]{
-					for (const auto& [fullName_, displayName_, _] : replayNames) {
+				int flags = ImGuiTableFlags_RowBg |
+					ImGuiTableFlags_BordersV |
+					ImGuiTableFlags_BordersOuter |
+					ImGuiTableFlags_Resizable |
+					ImGuiTableFlags_Sortable |
+					ImGuiTableFlags_Hideable |
+					ImGuiTableFlags_Reorderable |
+					ImGuiTableFlags_ContextMenuInBody |
+					ImGuiTableFlags_SizingStretchProp;
+				im::Table("##select-replay", 2, flags, {}, [&]{
```

Comment:
I understand this is still WIP.
Next steps are these?
* Add a table-header.
* Implement the sort functionality.

Reply:
Please see the commit message for some ideas on what's next :)

Reply:
OK, I couldn't resist and already added the headers and sorting functionality. But the code looks a bit clunky. Hints most welcome to improve it.